src/Lib/ImageTransform.php

```diff
@@ -27,6 +29,28 @@ class ImageTransform
     private $Imagine;
 
     /**
+     * @var Table $table Instance of the table being used
+     */
+    protected $Table;
```

Comment:
Generally speaking, better to use lowercase for all properties. I say that because since you are dropping the underscore for protected properties (cake version of PSR2 for now), then don't CamelCase for properties that hold objects (which is also a cake version of PSR2 until full adoption).